server/cache.ts

```diff
@@ -0,0 +1,12 @@
+interface RepoStarData {
+  starRecords: {
+    date: string;
+    count: number;
+  }[];
+  starAmount: number;
+}
+
+/**
+ * memory data cache
+ */
+export const repoStarDataCache = new Map<string, RepoStarData>();
```

Comment:
Need to comment on the cache behavior, what's the key composition of the cache and the expiring policy.

BTW, I didn't find where we expire the cache, so it will likely OOM after running for a while.